src/components/Toast/Toast.st.css

```diff
@@ -37,6 +37,13 @@
     background-color: value(statusColor);
 }
 
+.root:placement(bottomFullWidth) {
+    width: 100vw;
+    position: fixed;
+    bottom: 0px;
+    left: 0px;
+}
+
```

Comment:
I'm not sure about this prop.
Currently we let our consumers decide how to position the elements, this one as well.
Since we don't really know where the components are rendered.
So this is something we might need to think about.
In any case, I would add this styling for mobile only, i.e.:
```css
.root:mobile:placement(bottomFullWidth) { ... }
```

Reply:
it's up to you, I can write this styles in CopyUrlButton, but thought that it's could be useful to have this prop. Also, option could be to have <Notification/> which will extend Toast and add styles. But seems like Notification is the same as Toast, and no need to have separate component, idk. Other option to have <AbsoluteContainer position='bottom' />. It's up to you.